datahub-web-react/src/app/entity/shared/tabs/Dataset/Validations/fieldDescriptionUtils.ts

```diff
@@ -0,0 +1,179 @@
+import {
+    AssertionStdOperator,
+    AssertionStdParameters,
+    FieldAssertionInfo,
+    FieldAssertionType,
+    FieldMetricType,
+    FieldTransformType,
+} from '../../../../../../types.generated';
+import { formatNumberWithoutAbbreviation } from '../../../../../shared/formatNumber';
+import { parseMaybeStringAsFloatOrDefault } from '../../../../../shared/numberUtil';
+
+const ASSERTION_OPERATOR_TO_DESCRIPTION: Record<AssertionStdOperator, string | undefined> = {
```

Comment:
can you remind me which one of these were not working?

Reply:
AssertionStdOperator.Null,
    AssertionStdOperator.NotEqualTo,

Reply:
Yes.. It is Fixed